src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -103,14 +126,19 @@ class EventEntries extends React.Component {
     );
   }
 
-  recordIssueError(errorTypes, errorMessages) {
+  recordIssueError(errorTypes: any[], errorMessages: any[]) {
```

Comment:
I believe this can be an Array of string 
```suggestion
  recordIssueError(errorTypes: any[], errorMessages: string[]) {
```